src/config/HelixConfig.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const fs = require('fs-extra');
+const path = require('path');
+const yaml = require('js-yaml');
+const logger = require('../logger.js');
+const GitUrl = require('./GitUrl.js');
+const Strain = require('./Strain.js');
+
+const HELIX_CONFIG = 'helix-config.yaml';
+
+async function isFile(filePath) {
+  try {
+    return (await fs.stat(filePath)).isFile();
+  } catch (e) {
+    return false;
+  }
+}
+
+class HelixConfig {
+  constructor() {
+    this._cwd = process.cwd();
+    this._cfgPath = '';
+    this._cfgRelPath = '';
+    this._cfg = {};
+
+    this._defaults = {
+      content: new GitUrl('http://localhost/local/default/blob/master'),
+      code: new GitUrl('http://localhost/local/default/blob/master'),
+      staticRoot: '/dist',
+      directoryIndex: 'index.html',
```

Comment:
This is simply `index` in `strains.yaml`

Reply:
this is in the defaults. just using `index` outside of the context of a strain might be confusing.